### PR TITLE
Support eskip

### PR DESCRIPTION
**Pull request by danpersa**
Support the updated routes with eskip

### CODE REVIEW

File: src/it/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepoSpec.scala [hunk 1]
```diff
@@ -95,29 +95,28 @@ class RoutesPostgresRepoSpec extends FunSpec with BeforeAndAfter with Matchers w
...
         deleteRoute(1)
 
-        val routes: List[RouteRow] = routesRepo.selectModifiedSince(createdAt.minus(1, ChronoUnit.MICROS), LocalDateTime.now())
+        val routesWithPath: List[(RouteRow, PathRow)] =
+          routesRepo.selectModifiedSince(createdAt.minus(1, ChronoUnit.MICROS), LocalDateTime.now())
 
-        routes.size should be (3)
-        routes.map(_.id.get).toSet should be (Set(1, 3, 4))
+        routesWithPath.size should be (3)
+        routesWithPath.map(_._1.id.get).toSet should be (Set(1, 3, 4))
```
Comment (NorthFury) [inline src/it/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepoSpec.scala]: flatMap

File: src/main/scala/org/zalando/spearheads/innkeeper/services/EskipRouteService.scala [hunk 2]
```diff
@@ -1,28 +1,33 @@
...
-    })
+    routeAndPathRowsStreamToEskipStream{
+      routesRepo.selectModifiedSince(since, currentTime)
+    }
+  }
+
+  def currentEskipRoutes(currentTime: LocalDateTime = LocalDateTime.now()): Source[EskipRouteWrapper, NotUsed] = {
+    routeAndPathRowsStreamToEskipStream{
+      routesRepo.selectActiveRoutesWithPath(currentTime)
+    }
```
Comment (NorthFury) [inline src/main/scala/org/zalando/spearheads/innkeeper/services/EskipRouteService.scala]: This looks a bit funky but I can't figure out a better way of writing this. I mean `routeAndPathRowsStreamToEskipStream`.

**Comment by NorthFury** [comment]
:+1:

File: src/it/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepoSpec.scala (hunk 1 above)
Comment (danpersa) [inline src/it/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepoSpec.scala]: I still need a set

**Pull request merged**